test/run/docker-compose.spec.ts

```diff
@@ -424,14 +425,6 @@ describe("DockerCompose", () => {
                 once: mockOnce
             });
 
-            process.env = {
-                ...(process.env),
-                CHS_DEV_GITHUB_SSH_PRIVATE_KEY: sshPrivateKeyFile
-            };
-
-            readFileSyncMock.mockReturnValue(
-                Buffer.from(sshPrivateKey, "utf8")
-            );
```

Comment:
this not replaced by `existsSyncMock.mockReturnValue(true);`  ?

Reply:
So the configuration contains the value of the file now so no need to mock the env or readfileSyncMock response for it